lunatrace/bsl/frontend/src/components/sidebar/sidebarItems.tsx

```diff
@@ -11,17 +11,21 @@
  * limitations under the License.
  *
  */
-import { AlertOctagon, LogIn, Plus } from 'react-feather';
+import { AlertOctagon, LogIn, Plus, User } from 'react-feather';
 import { AiFillGithub } from 'react-icons/ai';
 import { BiUnlink } from 'react-icons/bi';
 import { RiParkingFill } from 'react-icons/ri';
 
-import { GetSidebarInfoQuery } from '../../api/generated';
+import { GetSidebarInfoQuery, Scalars } from '../../api/generated';
+import useAppSelector from '../../hooks/useAppSelector';
+import { userIsAdmin } from '../../store/slices/authentication';
 import { userHasAnyOrganizations } from '../../utils/organizations';
 
 import { NavSection, SidebarItem } from './types';
 
 export function generateSidebarItems(data: GetSidebarInfoQuery | undefined, isAuthenticated: boolean): NavSection[] {
+  const isAdmin = useAppSelector(userIsAdmin);
```

Comment:
We talked about getting this out of the store and instead just passing a boolean to this function `isAdmin`